server/controllers/CommentController.js

```diff
@@ -175,47 +175,49 @@ exports.reply = async (req, res) => {
   res.status(204).send();
 
   if (parent.userId !== "AITA") return;
-  // This is a reply to an AI generated comment.
-
-  console.log("Generating AI reply...");
-
-  const fetchedComments = await Comment.findAll({
-    where: {
-      solutionId: parent.solutionId,
-    },
-  });
-
-  const extracted = fetchedComments.map(
-    ({ id, text, userId, solutionId, upVotes, replies, usersWhoUpvoted }) => ({
-      id,
-      text,
-      userId,
-      solutionId,
-      hasUserUpvoted: usersWhoUpvoted.includes(req.user.username),
-      upVotes,
-      replies,
-    }),
-  );
-
-  const comments = formatComments(extracted);
-
-  // console.log(comments);
-
-  const comment_chain = comments
-    .map((c) => getCommentChain(c, comment.id))
-    .filter((ch) => ch.length !== 0)[0];
-  const solution = await Solution.findByPk(parent.solutionId);
-  const challenge = await Challenge.findByPk(solution.challengeId);
-
-  console.log("Calling AITA....");
-  const [chainRunId, feedback] = await AITA.feedback_for_comment(
-    comment_chain,
-    challenge,
-    solution,
-  );
 
-  await reply_to_comment(comment.id, feedback, "AITA", chainRunId);
-  console.log(`AITA commented on comment ${comment.id}`);
+  // UNCOMMENT TO ENABLE AITA COMMENT REPLIES
+  // // This is a reply to an AI generated comment.
+  //
+  // console.log("Generating AI reply...");
+  //
+  // const fetchedComments = await Comment.findAll({
+  //   where: {
+  //     solutionId: parent.solutionId,
+  //   },
+  // });
+  //
+  // const extracted = fetchedComments.map(
+  //   ({ id, text, userId, solutionId, upVotes, replies, usersWhoUpvoted }) => ({
+  //     id,
+  //     text,
+  //     userId,
+  //     solutionId,
+  //     hasUserUpvoted: usersWhoUpvoted.includes(req.user.username),
+  //     upVotes,
+  //     replies,
+  //   }),
+  // );
+  //
+  // const comments = formatComments(extracted);
+  //
+  // // console.log(comments);
+  //
+  // const comment_chain = comments
+  //   .map((c) => getCommentChain(c, comment.id))
+  //   .filter((ch) => ch.length !== 0)[0];
+  // const solution = await Solution.findByPk(parent.solutionId);
+  // const challenge = await Challenge.findByPk(solution.challengeId);
```

Comment:
Small thing, but maybe put the commented code in a function to make the code cleaner